### ISSUE TITLE
Inner Label word wrap

### ISSUE THREAD

**Issue by bweismann** [benkeen/d3pie#66]
I've got an inner label that would really be displayed better if I was able to place part of it on a second line, how can I achieve this?  I've tried \n escape but it doesn't work.  I think what I am looking for is similar to this but using a second label instead of percentage:
		"inner": {
			"format": "label-percentage2"
		},

![capture](https://cloud.githubusercontent.com/assets/11619821/6790033/bc55a37a-d172-11e4-8603-1f2b6411e10d.PNG)

**Comment by benkeen** [comment]
Agreed. I have a related ticket open for it: https://github.com/benkeen/d3pie/issues/16

With SVG it's trickier than you'd think, unfortunately. But I'll keep this in mind, thanks.

**Comment by philippecarle** [comment]
+1 ;)
This could be helpful maybe ? http://stackoverflow.com/questions/16701522/how-to-linebreak-an-svg-text-within-javascript

**Comment by inlikefletch** [comment]
+1
